easybuild/easyconfigs/p/Perl/Perl-5.32.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,1798 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'Perl'
+version = '5.32.1'
+
+homepage = 'https://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.cpan.org/src/%(version_major)s.0']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['03b693901cd8ae807231b1787798cf1f2e0b8a56218d07b7da44f784a7caeb2c']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('groff', '1.22.4'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),  # for Net::SSLeay
+    ('expat', '2.2.9'),  # for XML::Parser
+    ('ncurses', '6.2'),  # for Term::ReadLine::Gnu
+    ('libreadline', '8.1'),  # for Term::ReadLine::Gnu
+    ('DB', '18.1.40'),  # for DB_File
+]
+
+# OpenSSL is required for Net::SSLeay
+osdependencies = ['openssl', ('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Comment:
```suggestion
osdependencies = ['openssl', OS_PKG_OPENSSL_DEV]
```

Reply:
note to self: We can't actually add OpenSSL 1.1.1k as an alternative here.. because then we'd have a circular build dep (ouch). We have to rely on OS version for perl unless we split out modules from here. (lets not tackle that issue right now)